app/src/main/java/com/andrew/tasky/agenda/data/util/MapperFunctions.kt

```diff
@@ -5,16 +5,17 @@ import com.andrew.tasky.agenda.data.reminder.ReminderDto
 import com.andrew.tasky.agenda.data.reminder.ReminderEntity
 import com.andrew.tasky.agenda.data.task.TaskDto
 import com.andrew.tasky.agenda.domain.models.AgendaItem
+import com.andrew.tasky.agenda.domain.models.Attendee
+import com.andrew.tasky.agenda.domain.models.EventPhoto
 import java.time.LocalDateTime
 import java.time.ZoneOffset
 
 fun TaskDto.toTask(): AgendaItem.Task {
     return AgendaItem.Task(
         id = id,
-        isDone = isDone,
         title = title,
         description = description ?: "",
-        startDateAndTime = LocalDateTime.ofEpochSecond(time, 0, null),
+        startDateAndTime = LocalDateTime.ofEpochSecond(time, 0, ZoneOffset.UTC),
```

Comment:
If I'm not wrong, you want to use the local user's time zone here instead of `UTC` right? `LocalDateTime` does not contain any time zone information and should represent the local time as the name says. Alternatively, you can use `ZonedDateTime` which does contain this info :)